### PR TITLE
Added changes for PER ci tests and added new ME18 WLP_V1 board

### PR DESCRIPTION
**Pull request by yc-adi**

### CODE REVIEW

File: Libraries/Boards/MAX32690/WLP_V1/Source/board.c [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+#include "spixf.h"
+#include "tft_st7735.h"
+#include "uart.h"
+
+/***** Global Variables *****/
+mxc_uart_regs_t *ConsoleUart = MXC_UART_GET_UART(CONSOLE_UART);
+extern uint32_t SystemCoreClock;
+
+const mxc_gpio_cfg_t pb_pin[] = {
+    { MXC_GPIO4, MXC_GPIO_PIN_0, MXC_GPIO_FUNC_IN, MXC_GPIO_PAD_PULL_UP, MXC_GPIO_VSSEL_VDDIOH },
```
Comment (kevin-gillespie) [inline Libraries/Boards/MAX32690/WLP_V1/Source/board.c:54]: Remove hardware that the board doesn't have (external flash, tft, PBs...)

File: Libraries/Boards/MAX32690/WLP_V1/examples.txt [hunk 2]
```diff
@@ -0,0 +1,58 @@
+ADC
```
Comment (kevin-gillespie) [inline Libraries/Boards/MAX32690/WLP_V1/examples.txt:1]: We can probably limit this example to BLE5_ctr.

**Review by kevin-gillespie — DISMISSED** [review]

**Comment by sihyung-maxim** [comment]
Don't mind the failed SVD check. There's an issue with the workflow's token access for the forks. It's currently getting resolved.

**Comment by EdwinFairchild** [comment]
@yc-adi  did something change with the `w1` board?  Openocd cant find the device according the logs of failed BLE test. If I recall you left the name the same so it should require no code change

**Comment by yc-adi** [comment]
The new MAX32690 board was not power on. After power up, all tests except BLE_FreeRTOS were passed. @EdwinFairchild All BLE_FreeRTOS tests were failed.

**Review by EdwinFairchild — APPROVED** [review]

**Review by victor-levi-maxim — APPROVED** [review]

**Comment by yc-adi** [comment]
Will modify the WLP_V1 in the next pull request.

**Pull request merged**